backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -164,8 +165,17 @@ public boolean performNextOperation(int completedChildCount) {
 
         if (phase == VmBackupPhase.READY) {
             if (getParameters().getVmBackup().isStopped()) {
+                updateVmBackupPhase(VmBackupPhase.FINALIZING);
+            }
+            return true;
+        }
+
+        if (phase == VmBackupPhase.FINALIZING) {
+            if (!isAnyDiskLocked()) {
                 removeAutoGeneratedSnapshot(createStepsContext(StepEnum.MERGE_SNAPSHOTS), getActionType(), getParameters());
                 updateVmBackupPhase(VmBackupPhase.REMOVING_SNAPSHOT);
+            } else {
+                log.debug("Backup {} is waiting on disk to unlock", getParameters().getVmBackup().getVmId());
```

Comment:
so that would print 'Backup' followed by the VM identifier - need to change that to the backup identifier

Reply:
oops